depthai_ros_driver/src/pipelines/stereo_with_detection_pipeline.cpp

```diff
@@ -0,0 +1,134 @@
+#include <ros/console.h>
+
+#include <pluginlib/class_list_macros.h>
+#include <depthai_ros_driver/pipeline.hpp>
+#include <depthai_ros_driver/pipeline_common.hpp>
+
+#include <depthai/pipeline/node/ColorCamera.hpp>
+#include <depthai/pipeline/node/DetectionNetwork.hpp>
+#include <depthai/pipeline/node/MonoCamera.hpp>
+#include <depthai/pipeline/node/StereoDepth.hpp>
+#include <depthai/pipeline/node/XLinkOut.hpp>
+
+namespace depthai_ros_driver
+{
+class StereoWithDetectionPipeline : public rr::Pipeline {
```

Comment:
Thoughts on deriving from stereo and then adding just the nn part here?

Reply:
can't inherit when there's no header file?

Reply:
Not gonna be implemented in this commit. Will think about it since having header file to inherit stuff is definitely nice